src/aws/bin/terraform

```diff
@@ -2,6 +2,20 @@
 
 set -eo pipefail
 
+# Get current ngrok url from ngrok api launched with command "make ngrok"
+
+if docker-compose port ngrok 4040 &> /dev/null ; then
```

Comment:
Don't we need a mechanisme to make sure that we are not deploying this to production?
Check the terraform $workspace?

Reply:
I don't think this will be used in production, but I've changed how this works, and production should not be impacted.